### ISSUE TITLE
Dark Mode Manual

### ISSUE THREAD

**Issue by ccfman2004** [Yeriwyn/Salad_Fork#39]
While I appreciate the new extensive manual, unfortunately due to my eyesight, I cannot read anything that isn't in Dark Mode.  This is why I loved the web manual you had for v1.0 and why I opened the issue about the 404 error with the Docs.

I never liked Dark Mode until I got cataracts, now I can't see anything unless it's in Dark Mode.

**Comment by Yeriwyn** [comment]
Totally understandable -- Try this link directly to the gslides source document, maybe dark mode in your browser will work better than the pdf? I'm not sure if the images will convert properly or not, but it's worth a try

https://docs.google.com/presentation/d/1UCO_c2_HoszgfpVY2Mt0ubquJ9CTzp2s_0dCxSFrDjU/edit?usp=sharing

**Comment by ccfman2004** [comment]
Thanks. I may go and convert everything into a Dark Mode manually and do a PR.

**Comment by ccfman2004** [comment]
I've never used Google Slides before and I absolutely hate it. Why can't I just be able to right click an image to be able to save it just like basically every other program there is? Not being able to do this basic task is stopping me from being able convert the manual so I can actually read it.

**Comment by Yeriwyn** [comment]
I agree, there's a lot of basic usability missing in slides. It's the best of the worst options though, I tried a few other publishing apps in the beginning and they were all super difficult and frustrating to use
